.changelog/4158.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Updated the **validate** command (`BC106`) to verify that the fromversion property was not changed on existing Content files.
+  type: internal
+pr_number: 4158
```

Comment:
Remove this file, we don't announce new validations since the validate refactor is not officially out yet.

Reply:
Done.